### PR TITLE
Display final score on GUI's game over pop up

### PR DESCRIPTION
**Pull request by mshannahnv**

### CODE REVIEW

File: gui/src/main/java/DiceGameGUI.java [hunk 1]
```diff
@@ -282,6 +282,17 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
...
+                    null, 
+                    "Game Over! You save your score of " 
+                    + thisGame.getCurrentTotal()
+                    + "\nFinal Score: "
+                    + thisGame.getScore(),
+                    "Game Over!",
+                    JOptionPane.OK_OPTION);
+            btnRoll.setEnabled(false);
+            btnStop.setEnabled(false);
+
```
Comment (ngdao) [inline gui/src/main/java/DiceGameGUI.java:295]: this is have conflict with my code in #119, I like your version better. Just throw it in an if statement if(thisGame.getCurrentTotal <23)

**Comment by ngdao** [comment]
Everything beside my inline note is A-OK

File: gui/src/main/java/DiceGameGUI.java (hunk 1 above)
Comment (anderspitman) [inline gui/src/main/java/DiceGameGUI.java:295]: Someone will have to manually merge.

Comment (mshannahnv) [inline gui/src/main/java/DiceGameGUI.java:295]: @ngdao If they press the RESET button, their total would always be below 23, right?

Comment (ngdao) [inline gui/src/main/java/DiceGameGUI.java:295]: When they hit 23 or go over, after the game over message is displayed, they still need to hit reset, so another message is redundant

**Pull request closed without merging**